access/token_test.go

```diff
@@ -0,0 +1,66 @@
+package access_test
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/qri-io/qfs"
+	"github.com/qri-io/qri/access"
+	access_spec "github.com/qri-io/qri/access/spec"
+	cfgtest "github.com/qri-io/qri/config/test"
+	"github.com/qri-io/qri/repo/profile"
+)
+
+func TestPrivKeyTokens(t *testing.T) {
+	prevTs := access.Timestamp
+	access.Timestamp = func() time.Time { return time.Time{} }
+	defer func() { access.Timestamp = prevTs }()
+
+	peerInfo := cfgtest.GetTestPeerInfo(0)
+	tokens, err := access.NewPrivKeyTokenSource(peerInfo.PrivKey)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	pro := &profile.Profile{
+		ID:       profile.IDB58MustDecode(peerInfo.EncodedPeerID),
```

Comment:
Also here, use `peerInfo.PeerID` instead

Reply:
PeerID an ProfileID are not interchangeable here.